StressTest.ps1

```diff
@@ -0,0 +1,24 @@
+﻿[CmdletBinding()]
+param(
+    [string]
+    $BinSkimFolder = "bld\bin\x64_Release\netcoreapp3.1",
+
+    [string]
+    $OutputFolder = "c:\temp\",
+
+    [string]
+    $OutputFile = "binskim",
+
+    [string]
+    $InputPaths,
+
+    [int]
+    $Times = 1000
+)
+
+for ($i = 0; $i -lt $Times; $i++)
+{
+    $command = $BinSkimFolder + "\BinSkim.exe analyze --recurse --hashes --force --output " + $OutputFolder + $OutputFile + $i + ".sarif " + $InputFolder
+    Write-Host $command
+    Invoke-Expression $command
```

Comment:
>$command [](http://example.com/codeflow?start=22&length=8)

I think a special mode in this script where it breaks in the case that the output differs from previous run could be helpful.

Then stress could run indefinitely until there's a break.

Reply:
done.